packages/cli/src/ui/constants.ts

```diff
@@ -15,3 +15,5 @@ export const UI_WIDTH =
 export const STREAM_DEBOUNCE_MS = 100;
 
 export const SHELL_COMMAND_NAME = 'Shell Command';
+
+export const NARROW_BREAKPOINT = 80;
```

Comment:
nit: name NARROW_BREAKPOINT is a bit of a surprising name. Can we create a name that is a bit more idiomatic for how it is used. E.g. something like `isNarrowWidth(width:number)` might be a bit more idiomatic to me.